v3/src/components/app.tsx

```diff
@@ -91,17 +102,26 @@ export const App = observer(function App() {
     }
 
     async function initialize() {
+      const {sample, dashboard, di, noEntryModal} = urlParams
+      const _sample = sampleData.find(name => sample === name.toLowerCase())
+      const isDashboard = dashboard !== undefined
+      const hasHashFileParam = window.location.hash.startsWith("#file=examples:")
```

Comment:
This isn't going to catch all of the ways that the CFM can open a file, e.g. shared documents, etc. I think a better approach would be to close it upon receipt of `openedFile` from the CFM. One way to do that would be to add an `onFileOpened` callback to `useCloudFilemanager()` which is called when the `fileOpened` event is received.